README.md

```diff
@@ -1 +1,73 @@
-# Tic-Tac-Toe
\ No newline at end of file
+## TIC TAC TOE GAME
+
+
+# About the game
+
+The game involves two players each assigned a symbol X OR O. If any of the players has all his symbols aligned either horizontally,vertically ,diagnoally or antidiagnolly first then he/she becomes the winner. Otherwise, if the board is full and none of them has won then the game is a draw and thus game over.
+
+![Tic-tic-toe example](https://user-images.githubusercontent.com/53356820/104712392-c88b2980-5722-11eb-9b23-66bd332b769f.png)
+
+## Built With
+
+Ruby
+
+## Prerequisites
+
+- Ruby
+- Text editor eg. VS Code
+- Git.
+- Github profile
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.​
+
+1. Clone the repository
+`git clone https://github.com/keneogu/Tic-Tac-Toe.git`
+
+2. Change directory
+`cd Tic-Tac-Toe`
+
+3. Run the program
+`ruby bin/main.rb`
+
+4. Start the Game
```

Comment:
[Optional] Running `ruby bin/main.rb` automatically starts the game so you can combine steps 3 and 4 in one step: <br>
3. Start the game by running:
`ruby bin/main.rb`